pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/SkylineBatchConfigForm.cs

```diff
@@ -282,8 +290,11 @@ private void gridReportSettings_SelectionChanged(object sender, EventArgs e)
         private void SetInitialSkylineSettings(SkylineBatchConfig config)
         {
             if (!_canEditSkylineSettings) return;
+            if (config != null)
+                _skylineTypeControl = new SkylineTypeControl(config.UsesSkyline, config.UsesSkylineDaily, config.UsesCustomSkylinePath, config.SkylineSettings.CmdPath);
+            else
+                _skylineTypeControl = new SkylineTypeControl(false, false, true, "C:\\Program Files\\Skyline");

```

Comment:
It would also be good to have a comment here explaining when this can happen. It seems like you are defaulting to using a custom path pointing to C:\Program Files\Skyline regardless of whether it is there or not. That seems like it needs explaining.

Reply:
Added the comment. I realized I'm using a different constructor in my current version, so I replaced that as well. I'm trying not to change too much for the patch but I think this makes more sense.